package.json

```diff
@@ -1,22 +1,27 @@
 {
   "name": "jsonparse",
   "description": "This is a pure-js JSON streaming parser for node.js",
-  "tags": ["json", "stream"],
+  "tags": [
+    "json",
+    "stream"
+  ],
   "version": "1.3.1",
   "author": "Tim Caswell <tim@creationix.com>",
   "repository": {
     "type": "git",
     "url": "http://github.com/creationix/jsonparse.git"
   },
   "devDependencies": {
-    "tape": "~0.1.1",
+    "tape": "^4.9.0",
     "tap": "~0.3.3"
   },
   "scripts": {
     "test": "tap test/*.js"
   },
   "bugs": "http://github.com/creationix/jsonparse/issues",
-  "engines": ["node >= 0.2.0"],
+  "engines": [
+    "node >= 0.2.0"
```

Comment:
Did we look into this? I believe there was concern that optional parameters weren't supported in this really old node version?